libptable/ptable.c

```diff
@@ -157,12 +160,16 @@ static int ptable_verify(const ptable_t *ptable, uint32_t memsz, uint32_t blksz)
 
 int ptable_deserialize(ptable_t *ptable, uint32_t memsz, uint32_t blksz)
 {
+	int ret;
 	uint32_t i;
 
 	if (ptable == NULL) {
 		return -1;
 	}
 
+	/* CRC must be verified with data in little endian */
+	ret = ptable_verify(ptable, memsz, blksz);
+
```

Comment:
To avoid working on bad ptable (damaged `ptable->count` might crash the code)
```suggestion
	ret = ptable_verify(ptable, memsz, blksz);
	if (ret < 0) {
		return ret;
	}
```